Makefile

```diff
@@ -88,6 +89,19 @@ release-image:
 	@echo "Remember you need to push your image before calling make deploy"
 	@echo "    make push"
 
+debug-image: ## Build a docker image with the broker binary started in debug mode
+	env GOOS=linux go build -i -gcflags='-N -l' -o ${BUILD_DIR}/broker ./cmd/broker
+	env GOOS=linux go build -i -ldflags="-s -s" -o ${BUILD_DIR}/migration ./cmd/migration
+	env GOOS=linux go build -i -ldflags="-s -s" -o ${BUILD_DIR}/dashboard-redirector ./cmd/dashboard-redirector
+	docker build -f ${BUILD_DIR}/Dockerfile-debug -t ${BROKER_IMAGE}:${TAG} ${BUILD_DIR} --build-arg DEBUG_PORT=${ASB_DEBUG_PORT}
+	@./scripts/prep_debug_env.sh ${ASB_DEBUG_PORT}
+	@echo
+	@echo "Remember you need to push your image before updating deployment config"
+	@echo "    docker push ${BROKER_IMAGE}:${TAG}"
+	@echo
+	@echo "Run the following before connecting the debugger once the container is running"
+	@echo "oc port-forward <oc get po -o jsonpath='{.items[?(@.metadata.labels.service=="asb")].metadata.name}')> ${ASB_DEBUG_PORT}"
```

Comment:
add an `@echo ""` above the `oc port-forward` line also add 4 spaces in front of the `oc` command so that the output lines up with the `docker push` from above.

The current output looks like this:
```
Remember you need to push your image before updating deployment config
    docker push docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest

Run the following before connecting the debugger once the container is running
oc port-forward <oc get po -o jsonpath='{.items[?(@.metadata.labels.service==asb)].metadata.name}')> 9000
```

I believe this would look better, as it makes the commands more prominent:

```
Remember you need to push your image before updating deployment config

    docker push docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest

Run the following before connecting the debugger once the container is running

    oc port-forward <oc get po -o jsonpath='{.items[?(@.metadata.labels.service==asb)].metadata.name}')> 9000
```